File: Modul 1/Bahasa C (modul 1)/PRAK104-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
int main(){
    int a = 400000; 
    int b = 350000;
    float sum_a = a-(a*13/100) , sum_b = b-(b*21/100);

    printf("Harga sepatu a adalah %d\n", a);
    printf("Harga sepatu b adalah %d\n");
    printf("Sepatu A mendapatkan diskon 13%% sehingga harganya menjadi %.0f\n", sum_a);
    printf("Sepatu B mendapatkan diskon 21%% sehingga harganya menjadi %.0f", sum_b);

    return 0;
}

File: Modul 1/Bahasa python (modul 1)/PRAK104-2310817320015-AlfisyahRinaFajriati.py
a = 400000
b = 350000
da = 13
sda = a - (a * da // 100)
db = 21
sdb = b - (b * db // 100)
print("Harga sepatu A adalah", a)
print("Harga sepatu B adalah", b)
print(f"Sepatu A mendapat diskon {da}% sehingga harganya menjadi {sda}")
print(f"Sepatu B mendapat diskon {db}% sehingga harganya menjadi {sdb}")